### PR TITLE
Introduce Github Action workflows to run tests and coding style checks

### LINKED ISSUE #14: Use Github Actions instead of Travis?
Helpful library! Would you be open to transition to Github Actions and abandon Travis?

Travis not only got unreliable for Open Source projects (experienced it myself: tests weren't triggered after commits, not running for days), but may not be free anymore for them also.

Sources:
* https://www.nymedia.no/en/blog/why-we-decided-migrate-from-travis-to-github-action
* https://www.codetd.com/en/article/12116056

If you agree I can make a pull request and we can discuss details.

### PR DESCRIPTION
**Pull request by k00ni**
Fixes #14

PHPUnit 4 is a fixed dependency which results in problems running tests using higher PHP versions. I use your code in a PHP 8 project without any problems though.

It runs test for PHP 5.4 until 7.4. PHP 8 is ignored.

**Current status:** https://github.com/k00ni/php-jsonpointer/actions (see upper two entries)

### CODE REVIEW

**Comment by raphaelstolt** [comment]
Is it still work in progress or an actual pull request?

File: .github/workflows/CodingStyles.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                uses: actions/checkout@v2
+
+            -   name: Install PHP
+                uses: shivammathur/setup-php@v2
+
+            -   name: Install Composer dependencies
+                run: composer install --no-progress --no-suggest --prefer-dist --optimize-autoloader
+
+            -   name: Coding styles
+                run: composer run-script cs-fix
```
Comment (raphaelstolt) [inline .github/workflows/CodingStyles.yml:27]: Should be `composer run-script cs-lint`.

Comment (k00ni) [reply .github/workflows/CodingStyles.yml]: Done

**Comment by raphaelstolt** [comment]
Can you combine the commits into one i.e. `Improvement, replace Travis with GitHub Actions`.

**Comment by raphaelstolt** [comment]
Improvement, replace Travis with GitHub Actions

**Comment by raphaelstolt** [comment]
Thanks for this contribution.

**Pull request merged**